.travis-ocaml.sh

```diff
@@ -18,17 +18,34 @@ set -uex
 # the ocaml version to test
 OCAML_VERSION=${OCAML_VERSION:-latest}
 SYS_OCAML_VERSION=4.02
-OPAM_VERSION=${OPAM_VERSION:-2.0.0}
+# Default opam is the latest release of opam 2
+OPAM_VERSION=${OPAM_VERSION:-2}
 OPAM_INIT=${OPAM_INIT:-true}
 
+case $OPAM_VERSION in
+    2|2.0) OPAM_VERSION=2.0.1;;
```

Comment:
The idea is that this matrix will want updating as new releases of opam are made.